.github/workflows/dev-container.yml

```diff
@@ -384,7 +384,7 @@ jobs:
             # TODO: Consider using common top level directory for all artifacts: 'output/$TOOL/...'
             mkdir -pv pytest-results
             poetry --no-ansi --no-interaction show --tree
-            poetry --no-ansi --no-interaction run pytest --verbose --color=no | ts -i %.S | ts -s
+            poetry --no-ansi --no-interaction run pytest -n auto --verbose --color=no | ts -i %.S | ts -s
```

Comment:
I'm worried that it also should be capped at 4 here, but we can see on builds 1st.